source/sass/type.scss

```diff
@@ -278,3 +278,17 @@ a {
 
   @include scaleText(16px);
 }
+
+.opp-link {
+  font-family: Arvo;
+  font-size: 20px;
```

Comment:
Why the hard coded font size?

Reply:
Versus what?

Reply:
The `scaleText()` mixin used on previous lines

Reply:
Oh. Yeah I can add that.